pkg/providers/vsphere/validator.go

```diff
@@ -580,29 +587,46 @@ func (v *Validator) validateCPUserPrivs(ctx context.Context, spec *Spec, vuc *co
 		datacenter,
 	)
 	if err != nil {
-		return err
+		return false, err
 	}
 
 	return v.validatePrivs(ctx, privObjs, vsc)
 }
 
-func (v *Validator) validatePrivs(ctx context.Context, privObjs []PrivAssociation, vsc govmomi.VSphereClient) error {
-	var missingPrivs []string
+func (v *Validator) validatePrivs(ctx context.Context, privObjs []PrivAssociation, vsc govmomi.VSphereClient) (bool, error) {
+	var privs []string
 	var err error
+	missingPrivs := []missingPriv{}
+	passed := false
 
 	for _, obj := range privObjs {
 		path := obj.path
 		privsContent := obj.privsContent
 		t := obj.objectType
-		missingPrivs, err = v.getMissingPrivs(ctx, vsc, path, t, privsContent, vsc.Username())
+		privs, err = v.getMissingPrivs(ctx, vsc, path, t, privsContent, vsc.Username())
 		if err != nil {
-			return err
-		} else if len(missingPrivs) > 0 {
-			return fmt.Errorf("User %s missing privileges on %s: %v", vsc.Username(), path, missingPrivs)
+			return passed, err
+		} else if len(privs) > 0 {
+			missingPrivs = append(missingPrivs, missingPriv{
+				username:   vsc.Username(),
+				objectType: t,
+				path:       path,
+				privs:      privs,
+			})
 		}
 	}
 
-	return nil
+	if len(missingPrivs) == 0 {
+		passed = true
+	} else {
+		content, err := json.MarshalIndent(missingPrivs, " ", " ")
+		if err != nil {
+			return passed, nil
+		}
+		logger.MarkWarning(string(content)) // ToDo: this should be called in a top-level validation method
```

Comment:
What is the todo here still?

Reply:
How would the output look here?

Reply:
Ok, here is what I am imagining for user output. Let me know what you think.

At -v=0:
```
./cmd/eksctl-anywhere/main.go create cluster -f eksa-mgmt-cluster-2.yaml --force-cleanup
Warning: VSphereDatacenterConfig configured in insecure mode
Performing setup and validations
Warning: VSphereDatacenterConfig configured in insecure mode
✅ Connected to server
✅ Authenticated to vSphere
✅ Datacenter validated
✅ Network validated
✅ Datastore validated
✅ Folder validated
✅ Resource pool validated
✅ Datastore validated
✅ Folder validated
✅ Resource pool validated
✅ Control plane and Workload templates validated
⚠️ Warning: User jwmeiertest004@vsphere.local missing 190 vsphere permissions on /Datacenter/vm/jwmeier/permissiontest, cluster creation may fail. Re-run create cluster with --verbosity=3 to view specific missing permissions
⚠️ Warning: User jwmeiertest004@vsphere.local missing 190 vsphere permissions on /Datacenter/vm/Templates/bottlerocket-v1.22.10-kubernetes-1-22-eks-9-amd64-f18b278, cluster creation may fail. Re-run create cluster with --verbosity=3 to view specific missing permissions
⚠️ Warning: User jwmeiertest004@vsphere.local missing 190 vsphere permissions on /Datacenter/vm/Templates, cluster creation may fail. Re-run create cluster with --verbosity=3 to view specific missing permissions
✅ Vsphere Provider setup is valid
✅ Validate certificate for registry mirror
✅ Validate authentication for git provider
✅ Create preflight validations pass
Creating new bootstrap cluster
^CWarning: Terminating this operation may leave the cluster in an irrecoverable state
exit status 255
```

Reply:
And now at -v=3:
```
./cmd/eksctl-anywhere/main.go create cluster -f eksa-mgmt-cluster-2.yaml --force-cleanup -v=3
Warning: VSphereDatacenterConfig configured in insecure mode
Pulling docker image	{"image": "public.ecr.aws/l0g8r8j6/eks-anywhere-cli-tools:v0.11.1-eks-a-v0.0.0-dev-build.3855"}
Initializing long running container	{"name": "eksa_1661433788129129000", "image": "public.ecr.aws/l0g8r8j6/eks-anywhere-cli-tools:v0.11.1-eks-a-v0.0.0-dev-build.3855"}
Performing setup and validations
Warning: VSphereDatacenterConfig configured in insecure mode
✅ Connected to server
✅ Authenticated to vSphere
✅ Datacenter validated
✅ Network validated
Control plane VSphereMachineConfig template is not set. Using default template.
Template already exists. Skipping creation	{"template": "/Datacenter/vm/Templates/bottlerocket-v1.22.10-kubernetes-1-22-eks-9-amd64-f18b278"}
Control plane VSphereMachineConfig template is not set. Using default template.
Template already exists. Skipping creation	{"template": "/Datacenter/vm/Templates/bottlerocket-v1.22.10-kubernetes-1-22-eks-9-amd64-f18b278"}
✅ Datastore validated
✅ Folder validated
✅ Resource pool validated
✅ Datastore validated
✅ Folder validated
✅ Resource pool validated
✅ Control plane and Workload templates validated
⚠️  Warning: User jwmeiertest004@vsphere.local missing 190 vsphere permissions on /Datacenter/vm/jwmeier/permissiontest, cluster creation may fail. Re-run create cluster with --verbosity=3 to view specific missing permissions
⚠️  Warning: User jwmeiertest004@vsphere.local missing 190 vsphere permissions on /Datacenter/vm/Templates/bottlerocket-v1.22.10-kubernetes-1-22-eks-9-amd64-f18b278, cluster creation may fail. Re-run create cluster with --verbosity=3 to view specific missing permissions
⚠️  Warning: User jwmeiertest004@vsphere.local missing 190 vsphere permissions on /Datacenter/vm/Templates, cluster creation may fail. Re-run create cluster with --verbosity=3 to view specific missing permissions
Missing Permissions:
[{"Username":"jwmeiertest004@vsphere.local","objectType":"Folder","path":"/Datacenter/vm/jwmeier/permissiontest","Privs":["Alarm.Acknowledge","Alarm.Create","Alarm.Delete","Alarm.DisableActions","Alarm.Edit","Alarm.SetStatus","Authorization.ModifyPermissions","Authorization.ModifyRoles","CertificateManagement.Manage","Cns.Searchable","ComputePolicy.Manage","ContentLibrary.AddCertToTrustStore","ContentLibrary.CreateSubscribedLibrary","ContentLibrary.DeleteCertFromTrustStore","ContentLibrary.DeleteLibraryItem","ContentLibrary.DeleteLocalLibrary","ContentLibrary.DeleteSubscribedLibrary","ContentLibrary.DownloadSession","ContentLibrary.EvictLibraryItem","ContentLibrary.EvictSubscribedLibrary","ContentLibrary.GetConfiguration","ContentLibrary.ImportStorage","ContentLibrary.ProbeSubscription","ContentLibrary.ReadStorage","ContentLibrary.SyncLibrary","ContentLibrary.SyncLibraryItem","ContentLibrary.TypeIntrospection","ContentLibrary.UpdateConfiguration","ContentLibrary.UpdateLibrary","ContentLibrary.UpdateLibraryItem","ContentLibrary.UpdateLocalLibrary","ContentLibrary.UpdateSession","ContentLibrary.UpdateSubscribedLibrary","Datastore.AllocateSpace","Datastore.Browse","Datastore.Config","Datastore.DeleteFile","Datastore.FileManagement","Datastore.UpdateVirtualMachineFiles","Datastore.UpdateVirtualMachineMetadata","Extension.Register","Extension.Unregister","Extension.Update","Folder.Create","Folder.Delete","Folder.Move","Folder.Rename","Global.CancelTask","Global.GlobalTag","Global.Health","Global.LogEvent","Global.ManageCustomFields","Global.ServiceManagers","Global.SetCustomField","Global.SystemTag","HLM.Manage","Host.Hbr.HbrManagement","Namespaces.Configure","Namespaces.SelfServiceManage","Network.Assign","Resource.ApplyRecommendation","Resource.AssignVAppToPool","Resource.AssignVMToPool","Resource.ColdMigrate","Resource.CreatePool","Resource.DeletePool","Resource.EditPool","Resource.HotMigrate","Resource.MovePool","Resource.QueryVMotion","Resource.RenamePool","ScheduledTask.Create","ScheduledTask.Delete","ScheduledTask.Edit","ScheduledTask.Run","Sessions.GlobalMessage","Sessions.ValidateSession","StorageProfile.Update","StorageProfile.View","StorageViews.View","Trust.Manage","VApp.ApplicationConfig","VApp.AssignResourcePool","VApp.AssignVApp","VApp.AssignVM","VApp.Clone","VApp.Create","VApp.Delete","VApp.Export","VApp.ExtractOvfEnvironment","VApp.Import","VApp.InstanceConfig","VApp.ManagedByConfig","VApp.Move","VApp.PowerOff","VApp.PowerOn","VApp.Rename","VApp.ResourceConfig","VApp.Suspend","VApp.Unregister","VirtualMachine.Config.AddExistingDisk","VirtualMachine.Config.AddNewDisk","VirtualMachine.Config.AddRemoveDevice","VirtualMachine.Config.AdvancedConfig","VirtualMachine.Config.Annotation","VirtualMachine.Config.CPUCount","VirtualMachine.Config.ChangeTracking","VirtualMachine.Config.DiskExtend","VirtualMachine.Config.DiskLease","VirtualMachine.Config.EditDevice","VirtualMachine.Config.HostUSBDevice","VirtualMachine.Config.ManagedBy","VirtualMachine.Config.Memory","VirtualMachine.Config.MksControl","VirtualMachine.Config.QueryFTCompatibility","VirtualMachine.Config.QueryUnownedFiles","VirtualMachine.Config.RawDevice","VirtualMachine.Config.ReloadFromPath","VirtualMachine.Config.RemoveDisk","VirtualMachine.Config.Rename","VirtualMachine.Config.ResetGuestInfo","VirtualMachine.Config.Resource","VirtualMachine.Config.Settings","VirtualMachine.Config.SwapPlacement","VirtualMachine.Config.UpgradeVirtualHardware","VirtualMachine.GuestOperations.Execute","VirtualMachine.GuestOperations.Modify","VirtualMachine.GuestOperations.ModifyAliases","VirtualMachine.GuestOperations.Query","VirtualMachine.GuestOperations.QueryAliases","VirtualMachine.Hbr.ConfigureReplication","VirtualMachine.Hbr.MonitorReplication","VirtualMachine.Hbr.ReplicaManagement","VirtualMachine.Interact.AnswerQuestion","VirtualMachine.Interact.Backup","VirtualMachine.Interact.ConsoleInteract","VirtualMachine.Interact.CreateScreenshot","VirtualMachine.Interact.DefragmentAllDisks","VirtualMachine.Interact.DeviceConnection","VirtualMachine.Interact.DnD","VirtualMachine.Interact.GuestControl","VirtualMachine.Interact.Pause","VirtualMachine.Interact.PowerOff","VirtualMachine.Interact.PowerOn","VirtualMachine.Interact.PutUsbScanCodes","VirtualMachine.Interact.Reset","VirtualMachine.Interact.SESparseMaintenance","VirtualMachine.Interact.SetCDMedia","VirtualMachine.Interact.SetFloppyMedia","VirtualMachine.Interact.Suspend","VirtualMachine.Interact.ToolsInstall","VirtualMachine.Inventory.Create","VirtualMachine.Inventory.CreateFromExisting","VirtualMachine.Inventory.Delete","VirtualMachine.Inventory.Move","VirtualMachine.Inventory.Register","VirtualMachine.Inventory.Unregister","VirtualMachine.Namespace.Event","VirtualMachine.Namespace.EventNotify","VirtualMachine.Namespace.Management","VirtualMachine.Namespace.ModifyContent","VirtualMachine.Namespace.Query","VirtualMachine.Namespace.ReadContent","VirtualMachine.Provisioning.Clone","VirtualMachine.Provisioning.CloneTemplate","VirtualMachine.Provisioning.CreateTemplateFromVM","VirtualMachine.Provisioning.Customize","VirtualMachine.Provisioning.DeployTemplate","VirtualMachine.Provisioning.DiskRandomAccess","VirtualMachine.Provisioning.DiskRandomRead","VirtualMachine.Provisioning.FileRandomAccess","VirtualMachine.Provisioning.GetVmFiles","VirtualMachine.Provisioning.MarkAsTemplate","VirtualMachine.Provisioning.MarkAsVM","VirtualMachine.Provisioning.ModifyCustSpecs","VirtualMachine.Provisioning.PromoteDisks","VirtualMachine.Provisioning.PutVmFiles","VirtualMachine.Provisioning.ReadCustSpecs","VirtualMachine.State.CreateSnapshot","VirtualMachine.State.RemoveSnapshot","VirtualMachine.State.RenameSnapshot","VirtualMachine.State.RevertToSnapshot","VirtualMachineClasses.Manage","Vsan.Cluster.ShallowRekey","vService.CreateDependency","vService.DestroyDependency","vService.ReconfigureDependency","vService.UpdateDependency","vSphereDataProtection.Protection","vSphereDataProtection.Recovery"]},{"Username":"jwmeiertest004@vsphere.local","objectType":"VirtualMachine","path":"/Datacenter/vm/Templates/bottlerocket-v1.22.10-kubernetes-1-22-eks-9-amd64-f18b278","Privs":["Alarm.Acknowledge","Alarm.Create","Alarm.Delete","Alarm.DisableActions","Alarm.Edit","Alarm.SetStatus","Authorization.ModifyPermissions","Authorization.ModifyRoles","CertificateManagement.Manage","Cns.Searchable","ComputePolicy.Manage","ContentLibrary.AddCertToTrustStore","ContentLibrary.CreateSubscribedLibrary","ContentLibrary.DeleteCertFromTrustStore","ContentLibrary.DeleteLibraryItem","ContentLibrary.DeleteLocalLibrary","ContentLibrary.DeleteSubscribedLibrary","ContentLibrary.DownloadSession","ContentLibrary.EvictLibraryItem","ContentLibrary.EvictSubscribedLibrary","ContentLibrary.GetConfiguration","ContentLibrary.ImportStorage","ContentLibrary.ProbeSubscription","ContentLibrary.ReadStorage","ContentLibrary.SyncLibrary","ContentLibrary.SyncLibraryItem","ContentLibrary.TypeIntrospection","ContentLibrary.UpdateConfiguration","ContentLibrary.UpdateLibrary","ContentLibrary.UpdateLibraryItem","ContentLibrary.UpdateLocalLibrary","ContentLibrary.UpdateSession","ContentLibrary.UpdateSubscribedLibrary","Datastore.AllocateSpace","Datastore.Browse","Datastore.Config","Datastore.DeleteFile","Datastore.FileManagement","Datastore.UpdateVirtualMachineFiles","Datastore.UpdateVirtualMachineMetadata","Extension.Register","Extension.Unregister","Extension.Update","Folder.Create","Folder.Delete","Folder.Move","Folder.Rename","Global.CancelTask","Global.GlobalTag","Global.Health","Global.LogEvent","Global.ManageCustomFields","Global.ServiceManagers","Global.SetCustomField","Global.SystemTag","HLM.Manage","Host.Hbr.HbrManagement","Namespaces.Configure","Namespaces.SelfServiceManage","Network.Assign","Resource.ApplyRecommendation","Resource.AssignVAppToPool","Resource.AssignVMToPool","Resource.ColdMigrate","Resource.CreatePool","Resource.DeletePool","Resource.EditPool","Resource.HotMigrate","Resource.MovePool","Resource.QueryVMotion","Resource.RenamePool","ScheduledTask.Create","ScheduledTask.Delete","ScheduledTask.Edit","ScheduledTask.Run","Sessions.GlobalMessage","Sessions.ValidateSession","StorageProfile.Update","StorageProfile.View","StorageViews.View","Trust.Manage","VApp.ApplicationConfig","VApp.AssignResourcePool","VApp.AssignVApp","VApp.AssignVM","VApp.Clone","VApp.Create","VApp.Delete","VApp.Export","VApp.ExtractOvfEnvironment","VApp.Import","VApp.InstanceConfig","VApp.ManagedByConfig","VApp.Move","VApp.PowerOff","VApp.PowerOn","VApp.Rename","VApp.ResourceConfig","VApp.Suspend","VApp.Unregister","VirtualMachine.Config.AddExistingDisk","VirtualMachine.Config.AddNewDisk","VirtualMachine.Config.AddRemoveDevice","VirtualMachine.Config.AdvancedConfig","VirtualMachine.Config.Annotation","VirtualMachine.Config.CPUCount","VirtualMachine.Config.ChangeTracking","VirtualMachine.Config.DiskExtend","VirtualMachine.Config.DiskLease","VirtualMachine.Config.EditDevice","VirtualMachine.Config.HostUSBDevice","VirtualMachine.Config.ManagedBy","VirtualMachine.Config.Memory","VirtualMachine.Config.MksControl","VirtualMachine.Config.QueryFTCompatibility","VirtualMachine.Config.QueryUnownedFiles","VirtualMachine.Config.RawDevice","VirtualMachine.Config.ReloadFromPath","VirtualMachine.Config.RemoveDisk","VirtualMachine.Config.Rename","VirtualMachine.Config.ResetGuestInfo","VirtualMachine.Config.Resource","VirtualMachine.Config.Settings","VirtualMachine.Config.SwapPlacement","VirtualMachine.Config.UpgradeVirtualHardware","VirtualMachine.GuestOperations.Execute","VirtualMachine.GuestOperations.Modify","VirtualMachine.GuestOperations.ModifyAliases","VirtualMachine.GuestOperations.Query","VirtualMachine.GuestOperations.QueryAliases","VirtualMachine.Hbr.ConfigureReplication","VirtualMachine.Hbr.MonitorReplication","VirtualMachine.Hbr.ReplicaManagement","VirtualMachine.Interact.AnswerQuestion","VirtualMachine.Interact.Backup","VirtualMachine.Interact.ConsoleInteract","VirtualMachine.Interact.CreateScreenshot","VirtualMachine.Interact.DefragmentAllDisks","VirtualMachine.Interact.DeviceConnection","VirtualMachine.Interact.DnD","VirtualMachine.Interact.GuestControl","VirtualMachine.Interact.Pause","VirtualMachine.Interact.PowerOff","VirtualMachine.Interact.PowerOn","VirtualMachine.Interact.PutUsbScanCodes","VirtualMachine.Interact.Reset","VirtualMachine.Interact.SESparseMaintenance","VirtualMachine.Interact.SetCDMedia","VirtualMachine.Interact.SetFloppyMedia","VirtualMachine.Interact.Suspend","VirtualMachine.Interact.ToolsInstall","VirtualMachine.Inventory.Create","VirtualMachine.Inventory.CreateFromExisting","VirtualMachine.Inventory.Delete","VirtualMachine.Inventory.Move","VirtualMachine.Inventory.Register","VirtualMachine.Inventory.Unregister","VirtualMachine.Namespace.Event","VirtualMachine.Namespace.EventNotify","VirtualMachine.Namespace.Management","VirtualMachine.Namespace.ModifyContent","VirtualMachine.Namespace.Query","VirtualMachine.Namespace.ReadContent","VirtualMachine.Provisioning.Clone","VirtualMachine.Provisioning.CloneTemplate","VirtualMachine.Provisioning.CreateTemplateFromVM","VirtualMachine.Provisioning.Customize","VirtualMachine.Provisioning.DeployTemplate","VirtualMachine.Provisioning.DiskRandomAccess","VirtualMachine.Provisioning.DiskRandomRead","VirtualMachine.Provisioning.FileRandomAccess","VirtualMachine.Provisioning.GetVmFiles","VirtualMachine.Provisioning.MarkAsTemplate","VirtualMachine.Provisioning.MarkAsVM","VirtualMachine.Provisioning.ModifyCustSpecs","VirtualMachine.Provisioning.PromoteDisks","VirtualMachine.Provisioning.PutVmFiles","VirtualMachine.Provisioning.ReadCustSpecs","VirtualMachine.State.CreateSnapshot","VirtualMachine.State.RemoveSnapshot","VirtualMachine.State.RenameSnapshot","VirtualMachine.State.RevertToSnapshot","VirtualMachineClasses.Manage","Vsan.Cluster.ShallowRekey","vService.CreateDependency","vService.DestroyDependency","vService.ReconfigureDependency","vService.UpdateDependency","vSphereDataProtection.Protection","vSphereDataProtection.Recovery"]},{"Username":"jwmeiertest004@vsphere.local","objectType":"Folder","path":"/Datacenter/vm/Templates","Privs":["Alarm.Acknowledge","Alarm.Create","Alarm.Delete","Alarm.DisableActions","Alarm.Edit","Alarm.SetStatus","Authorization.ModifyPermissions","Authorization.ModifyRoles","CertificateManagement.Manage","Cns.Searchable","ComputePolicy.Manage","ContentLibrary.AddCertToTrustStore","ContentLibrary.CreateSubscribedLibrary","ContentLibrary.DeleteCertFromTrustStore","ContentLibrary.DeleteLibraryItem","ContentLibrary.DeleteLocalLibrary","ContentLibrary.DeleteSubscribedLibrary","ContentLibrary.DownloadSession","ContentLibrary.EvictLibraryItem","ContentLibrary.EvictSubscribedLibrary","ContentLibrary.GetConfiguration","ContentLibrary.ImportStorage","ContentLibrary.ProbeSubscription","ContentLibrary.ReadStorage","ContentLibrary.SyncLibrary","ContentLibrary.SyncLibraryItem","ContentLibrary.TypeIntrospection","ContentLibrary.UpdateConfiguration","ContentLibrary.UpdateLibrary","ContentLibrary.UpdateLibraryItem","ContentLibrary.UpdateLocalLibrary","ContentLibrary.UpdateSession","ContentLibrary.UpdateSubscribedLibrary","Datastore.AllocateSpace","Datastore.Browse","Datastore.Config","Datastore.DeleteFile","Datastore.FileManagement","Datastore.UpdateVirtualMachineFiles","Datastore.UpdateVirtualMachineMetadata","Extension.Register","Extension.Unregister","Extension.Update","Folder.Create","Folder.Delete","Folder.Move","Folder.Rename","Global.CancelTask","Global.GlobalTag","Global.Health","Global.LogEvent","Global.ManageCustomFields","Global.ServiceManagers","Global.SetCustomField","Global.SystemTag","HLM.Manage","Host.Hbr.HbrManagement","Namespaces.Configure","Namespaces.SelfServiceManage","Network.Assign","Resource.ApplyRecommendation","Resource.AssignVAppToPool","Resource.AssignVMToPool","Resource.ColdMigrate","Resource.CreatePool","Resource.DeletePool","Resource.EditPool","Resource.HotMigrate","Resource.MovePool","Resource.QueryVMotion","Resource.RenamePool","ScheduledTask.Create","ScheduledTask.Delete","ScheduledTask.Edit","ScheduledTask.Run","Sessions.GlobalMessage","Sessions.ValidateSession","StorageProfile.Update","StorageProfile.View","StorageViews.View","Trust.Manage","VApp.ApplicationConfig","VApp.AssignResourcePool","VApp.AssignVApp","VApp.AssignVM","VApp.Clone","VApp.Create","VApp.Delete","VApp.Export","VApp.ExtractOvfEnvironment","VApp.Import","VApp.InstanceConfig","VApp.ManagedByConfig","VApp.Move","VApp.PowerOff","VApp.PowerOn","VApp.Rename","VApp.ResourceConfig","VApp.Suspend","VApp.Unregister","VirtualMachine.Config.AddExistingDisk","VirtualMachine.Config.AddNewDisk","VirtualMachine.Config.AddRemoveDevice","VirtualMachine.Config.AdvancedConfig","VirtualMachine.Config.Annotation","VirtualMachine.Config.CPUCount","VirtualMachine.Config.ChangeTracking","VirtualMachine.Config.DiskExtend","VirtualMachine.Config.DiskLease","VirtualMachine.Config.EditDevice","VirtualMachine.Config.HostUSBDevice","VirtualMachine.Config.ManagedBy","VirtualMachine.Config.Memory","VirtualMachine.Config.MksControl","VirtualMachine.Config.QueryFTCompatibility","VirtualMachine.Config.QueryUnownedFiles","VirtualMachine.Config.RawDevice","VirtualMachine.Config.ReloadFromPath","VirtualMachine.Config.RemoveDisk","VirtualMachine.Config.Rename","VirtualMachine.Config.ResetGuestInfo","VirtualMachine.Config.Resource","VirtualMachine.Config.Settings","VirtualMachine.Config.SwapPlacement","VirtualMachine.Config.UpgradeVirtualHardware","VirtualMachine.GuestOperations.Execute","VirtualMachine.GuestOperations.Modify","VirtualMachine.GuestOperations.ModifyAliases","VirtualMachine.GuestOperations.Query","VirtualMachine.GuestOperations.QueryAliases","VirtualMachine.Hbr.ConfigureReplication","VirtualMachine.Hbr.MonitorReplication","VirtualMachine.Hbr.ReplicaManagement","VirtualMachine.Interact.AnswerQuestion","VirtualMachine.Interact.Backup","VirtualMachine.Interact.ConsoleInteract","VirtualMachine.Interact.CreateScreenshot","VirtualMachine.Interact.DefragmentAllDisks","VirtualMachine.Interact.DeviceConnection","VirtualMachine.Interact.DnD","VirtualMachine.Interact.GuestControl","VirtualMachine.Interact.Pause","VirtualMachine.Interact.PowerOff","VirtualMachine.Interact.PowerOn","VirtualMachine.Interact.PutUsbScanCodes","VirtualMachine.Interact.Reset","VirtualMachine.Interact.SESparseMaintenance","VirtualMachine.Interact.SetCDMedia","VirtualMachine.Interact.SetFloppyMedia","VirtualMachine.Interact.Suspend","VirtualMachine.Interact.ToolsInstall","VirtualMachine.Inventory.Create","VirtualMachine.Inventory.CreateFromExisting","VirtualMachine.Inventory.Delete","VirtualMachine.Inventory.Move","VirtualMachine.Inventory.Register","VirtualMachine.Inventory.Unregister","VirtualMachine.Namespace.Event","VirtualMachine.Namespace.EventNotify","VirtualMachine.Namespace.Management","VirtualMachine.Namespace.ModifyContent","VirtualMachine.Namespace.Query","VirtualMachine.Namespace.ReadContent","VirtualMachine.Provisioning.Clone","VirtualMachine.Provisioning.CloneTemplate","VirtualMachine.Provisioning.CreateTemplateFromVM","VirtualMachine.Provisioning.Customize","VirtualMachine.Provisioning.DeployTemplate","VirtualMachine.Provisioning.DiskRandomAccess","VirtualMachine.Provisioning.DiskRandomRead","VirtualMachine.Provisioning.FileRandomAccess","VirtualMachine.Provisioning.GetVmFiles","VirtualMachine.Provisioning.MarkAsTemplate","VirtualMachine.Provisioning.MarkAsVM","VirtualMachine.Provisioning.ModifyCustSpecs","VirtualMachine.Provisioning.PromoteDisks","VirtualMachine.Provisioning.PutVmFiles","VirtualMachine.Provisioning.ReadCustSpecs","VirtualMachine.State.CreateSnapshot","VirtualMachine.State.RemoveSnapshot","VirtualMachine.State.RenameSnapshot","VirtualMachine.State.RevertToSnapshot","VirtualMachineClasses.Manage","Vsan.Cluster.ShallowRekey","vService.CreateDependency","vService.DestroyDependency","vService.ReconfigureDependency","vService.UpdateDependency","vSphereDataProtection.Protection","vSphereDataProtection.Recovery"]}]
✅ Vsphere Provider setup is valid
✅ Validate certificate for registry mirror
✅ Validate authentication for git provider
✅ Create preflight validations pass
Creating new bootstrap cluster
Provider specific pre-capi-install-setup on bootstrap cluster
Installing cluster-api providers on bootstrap cluster
^CWarning: Terminating this operation may leave the cluster in an irrecoverable state
exit status 255
```